### PR TITLE
impact section removed

### PR DESCRIPTION
**Pull request by Veenakrishna2313**
# Description: Removed the Impact section from all components using it.

[comment]: # 'Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context.'

Fixes # (issue)

## Type of change

[comment]: # 'Please delete options that are not relevant.'

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Screenshots

[comment]: # 'Please include screenshots of your changes if relevant.'

|       Before        |       After        |
| :-----------------: | :----------------: |
| "screenshot before" | "screenshot after" |

# How Has This Been Tested?

- [ ] Cypress integration
- [ ] Cypress component tests
- [ ] Jest unit tests

# Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/pages/organizations/[organizationid].js [hunk 1]
```diff
@@ -463,7 +463,7 @@ export default function Organization(props) {
               mt: [10, 20],
             }}
           />
-          <ImpactSection />
+          {/**<ImpactSection />  */}
```
Comment (dadiorchen) [inline src/pages/organizations/[organizationid].js:466]: @Veenakrishna2313 please simply delete the code, a clear codebase is more important than considering some reuse of code in the future.

Comment (Veenakrishna2313) [reply src/pages/organizations/[organizationid].js]: I have deleted the code

**Comment by dadiorchen** [comment]
@Veenakrishna2313 thank you!

**Comment by RubenSmn** [comment]
@Veenakrishna2313 could you cleanup the unused imports as well?

**Pull request merged**